src/main/java/net/imglib2/algorithm/gauss3/Gauss3.java

```diff
@@ -126,10 +127,9 @@
 	 */
 	public static < S extends NumericType< S >, T extends NumericType< T > > void gauss( final double[] sigma, final RandomAccessible< S > source, final RandomAccessibleInterval< T > target ) throws IncompatibleTypeException
 	{
-		final int numthreads = Runtime.getRuntime().availableProcessors();
```

Comment:
Please update the javadoc:
"Computation is multi-threaded with as many threads as processors available."
should be something like
"...Computation may be multi-threaded, according to the current {@link Parallelization} context. (By default, runs on the common ForkJoinPool ...)"

Reply:
...and the same comment change should be added to the
`gauss( final double sigma, final RandomAccessible< S > source, final RandomAccessibleInterval< T > target )`
variant

Reply:
Good point. I updated the javadoc.